provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -1344,6 +1344,33 @@
         "environment",
         "permission"
       ]
+    },
+    "pulumiservice:index:EnvironmentVersionTag": {
```

Comment:
Does the resource not have any outputs?

Reply:
We should mirror all inputs as outputs, at least

Reply:
Good catch, somehow works without it lol
I'll add real quick